app/javascript/packs/list.jsx

```diff
@@ -38,21 +40,37 @@ class List extends React.Component {
         return -1;
       }
     });
+
     const projects = this.state.projects;
     const projectsList = projects.map(project => {
+      const ProjectUser = project.users.map(user => {
```

Comment:
`numberList` is now very long and has multiple responsibilities. You should split it